<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>177</height>
   </rect>
  </property>
  <property name="mouseTracking" >
   <bool>true</bool>
  </property>
  <property name="windowTitle" >
   <string>Settings Manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="font" >
      <font>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="styleSheet" >
      <string notr="true" />
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text" >
      <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:11pt; font-weight:600; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" color:#7777b1;">&amp;amp;User : &lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="buddy" >
      <cstring>userCombo</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QComboBox" name="userCombo" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="focusPolicy" >
      <enum>Qt::StrongFocus</enum>
     </property>
     <item>
      <property name="text" >
       <string>Staff</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Admin</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="font" >
      <font>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <stylestrategy>PreferDefault</stylestrategy>
      </font>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text" >
      <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:11pt; font-weight:600; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" color:#6a6a9e;">&amp;amp;Password :   &lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
     </property>
     <property name="buddy" >
      <cstring>passwordLe</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="passwordLe" >
     <property name="font" >
      <font>
       <stylestrategy>PreferAntialias</stylestrategy>
      </font>
     </property>
     <property name="echoMode" >
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <spacer name="horizontalSpacer_4" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QGroupBox" name="groupBox2" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string/>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="2" column="0" colspan="2" >
       <widget class="QLabel" name="label_3" >
        <property name="font" >
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text" >
         <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:600; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" color:#55aa00;">&amp;amp;Enter New Password&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="buddy" >
         <cstring>le1</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_4" >
        <property name="font" >
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text" >
         <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:600; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" color:#55aa00;">&amp;amp;Again&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="buddy" >
         <cstring>le2</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <spacer name="horizontalSpacer_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1" colspan="2" >
       <spacer name="horizontalSpacer_5" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="6" column="3" >
       <widget class="QPushButton" name="cancelButton" >
        <property name="text" >
         <string>&amp;Cancel</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2" >
       <widget class="QPushButton" name="saveButton" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="text" >
         <string>&amp;Save</string>
        </property>
        <property name="autoDefault" >
         <bool>true</bool>
        </property>
        <property name="default" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3" >
       <widget class="QLineEdit" name="le1" >
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="3" >
       <widget class="QLineEdit" name="le2" >
        <property name="echoMode" >
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>userCombo</tabstop>
  <tabstop>passwordLe</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>userCombo</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>passwordLe</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>196</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel" >
     <x>195</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>le2</sender>
   <signal>returnPressed()</signal>
   <receiver>saveButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>274</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel" >
     <x>134</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
